src/index.css

```diff
@@ -62,3 +62,19 @@ a,
   margin-left: 0.5em;
   margin-right: 0;
 }
+
+.screenReaderOnly {
+  /* TLDR: The [aria-label] attribute isn’t announced consistently 
+  If text is not needed for sighted users, hide it this way. 
+  https://gomakethings.com/revisting-aria-label-versus-a-visually-hidden-class/#:~:text=In%20the%20posts%2C%20I%20recommended,visually%20shown%20to%20sighted%20users. */
+  border: 0;
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  display: inline-block;
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  width: 1px;
+}
```

Comment:
Note that this CSS is not included in the published npm package. It is only used in the local dev site (`yarn start`).

Reply:
Thank you for catching that! I've relocated this utility in https://github.com/VEuPathDB/web-eda/pull/1613/commits/d82b03707176bcb81959773e08a0a863913d771e.

Reply:
> Do you think it would be worth wiring the filter chips with the appState subsetVariableAndEntity and isSubsetPanelOpen properties?

I see. These would be the ingredients needed to achieve the following: 
![chips](https://user-images.githubusercontent.com/24446573/220743326-273fc744-1f5e-41b8-aba6-e57d32c7b305.gif)

I wasn't sure if we wanted that in this unit of work or in this one: https://github.com/VEuPathDB/web-eda/issues/1498.

I'm fine with either, but do I understand you correctly?

Reply:
Awesome! :rocket: